tsconfig.json

```diff
@@ -18,8 +18,11 @@
         "resolveJsonModule": true
     },
     "compileOnSave": true,
+    "exclude": ["node_modules", "dist"],
     "includes": [
         "main.ts",
-        "globals.d.ts"
+        "globals.d.ts",
+        "lib",
+        "tests"
```

Comment:
I think tests should not be included in the compiled project

I'm not sure why this change, I guess something to do with eslint detecting what to lint?

Reply:
ah yeh. I was having trouble with getting eslint to lint files from anywhere except the root dir, and i added this while debugging. It's not required so i've put it back to how it was.